### ISSUE TITLE
ESLint wants successive indentation for chained ternary

### ISSUE THREAD

**Issue by fatcerberus** [eslint/eslint#10086]
When writing a chained ternary expression of the form shown below, ESLint wants successive indent levels for each link in the chain, which quickly gets out of hand for long chains.

**Tell us about your environment**

* **ESLint Version:** 4.18.2
* **Node Version:** 9.8.0
* **npm Version:** 5.6.0

**What parser (default, Babel-ESLint, etc.) are you using?**
Default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
env:
  es6: true
extends: 'eslint:recommended'
parserOptions:
  ecmaVersion: 8
  sourceType: module
globals:
  global: true
  console: true
  require: true
  Sphere: true
  Color: true
  DirectoryStream: true
  Dispatch: true
  FS: true
  FileOp: true
  FileStream: true
  Font: true
  IndexList: true
  Joystick: true
  Key: true
  Keyboard: true
  Mixer: true
  Model: true
  Mouse: true
  MouseKey: true
  RNG: true
  SSj: true
  Sample: true
  Server: true
  Shader: true
  Shape: true
  ShapeType: true
  Socket: true
  Sound: true
  SoundStream: true
  Surface: true
  TextDecoder: true
  TextEncoder: true
  Texture: true
  Transform: true
  VertexList: true
rules:
  array-bracket-spacing:
    - error
    - always
  comma-dangle:
    - error
    - always-multiline
  comma-spacing: error
  complexity:
    - warn
  func-style:
    - error
    - declaration
    - {
        allowArrowFunctions: true
      }
  indent:
    - error
    - tab
    - {
        SwitchCase: 1
      }
  no-console: off
  no-floating-decimal: error
  no-implicit-coercion: error
  no-invalid-this: error
  no-lone-blocks: error
  no-unused-vars:
    - error
    - {
        args: none
      }
  object-curly-spacing:
    - error
    - always
  semi:
    - error
    - always
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
214		let targets = this.targets !== null ? this.targets
215			: targetUnit !== null ? [ targetUnit ]
216			: skillToUse.defaultTargets(this.unit);
```

**What did you expect to happen?**
No trigger of `indent` rule

**What actually happened? Please include the actual, raw output from ESLint.**
`indent` rule triggered:
```
D:\src\spectacles-i\src\battleSystem\autoBattler.mjs
  200:1  error  Expected indentation of 4 tabs but found 3  indent
  216:1  error  Expected indentation of 4 tabs but found 3  indent

✖ 2 problems (2 errors, 0 warnings)
  2 errors, 0 warnings potentially fixable with the `--fix` option.
```

**Comment by not-an-aardvark** [comment]
Thanks for the report. However, this is working as intended; you can use the [`flatTernaryExpressions`](https://eslint.org/docs/rules/indent#flatternaryexpressions) option to change this behavior.

**Comment by fatcerberus** [comment]
> you can use the `flatTernaryExpressions` option to change this behavior.

Oof, how did I miss that? 🤦‍♂️   Thanks!
